iac-extensions/arm/src/main/java/org/sonar/iac/arm/parser/bicep/BicepLexicalGrammar.java

```diff
@@ -138,7 +141,7 @@ private static void lexical(LexerlessGrammarBuilder b) {
     b.rule(QUOTED_STRING_LITERAL).is(SPACING, b.regexp(BicepLexicalConstant.QUOTED_STRING_LITERAL_NO_QUOTES));
     b.rule(ALPHA_NUMERAL_STRING).is(SPACING, b.regexp(BicepLexicalConstant.ALPHA_NUMERAL_STRING));
     b.rule(STRING_LITERAL_VALUE).is(SPACING, b.regexp(BicepLexicalConstant.STRING));
-    b.rule(MULTILINE_STRING_REGEX).is(SPACING, b.regexp(BicepLexicalConstant.MULTILINE_STRING));
+    b.rule(MULTILINE_STRING_REGEX).is(SPACING, b.regexp(BicepLexicalConstant.MULTILINE_STRING_CONTENT));
```

Comment:
Why this is called REGEX and not CONTENT? Introducing too many words can lead to confusion.

Reply:
Sharing the same name also can lead to confusion: https://github.com/SonarSource/sonar-iac/pull/803#discussion_r1259670968

Reply:
That's true. However, adding the suffix `regex` looks odd to me, because the other rules are also based on regex and not extended by the suffix.